lib/curl_addrinfo.c

```diff
@@ -50,10 +50,13 @@
 #  define in_addr_t unsigned long
 #endif
 
-#if defined(USE_UNIX_SOCKETS) && defined(WINAPI_FAMILY) && \
-    (WINAPI_FAMILY == WINAPI_FAMILY_APP)
+#if defined(USE_UNIX_SOCKETS)
+#if defined(WINAPI_FAMILY) && (WINAPI_FAMILY == WINAPI_FAMILY_APP)
    /* Required for sockaddr_un type */
 #  include <afunix.h>
```

Comment:
I think these could be removed, the uwp build issue (#7006 ) can also be fixed by unixsockets_win32.h, but I'm not very sure.